src/app/_shared/landing/landing.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component } from '@angular/core';
-import {NavbarComponent} from "../core/layout/navbar/navbar.component";
+import {NavbarComponent} from "../../_teacher/navbar/navbar.component";
```

Comment:
Why _ ?

Reply:
Arch was crap had to improve and make some arrangements but i ain't done yet